bundles/org.openhab.binding.ekey/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -41,6 +41,11 @@
 				<description>@text/thing-type.config.ekey.delimiter.description</description>
 				<default>_</default>
 			</parameter>
+			<parameter name="sourceIp" type="text" required="false">
+				<context>network-address</context>
+				<label>@text/thing-type.config.ekey.sourceIp.label</label>
+				<description>@text/thing-type.config.ekey.sourceIp.description</description>
```

Comment:
Wouldn't it be better to call this natIP instead of sourceIp ? In fact, it is not the source of the packets.

Reply:
Done